mattermost/notifier.go

```diff
@@ -26,12 +25,19 @@ func (n Notifier) Notify(msg string) error {
 		bytes.NewBuffer([]byte(`{"text":"`+msg+`"}`)),
 	)
 	if err != nil {
-		return fmt.Errorf("notify: %v", err)
+		return fmt.Errorf("notify: host=%s %w", n.host(), err)
```

Comment:
Hi @kahlys 

Reasons to prefer logging only the hostname rather than the endpoint:
1. The logs should not reveal secrets, the URL contains a secret token
2. The log is more human readable with a short hostname rather than a long URL endpoint
3. Most of the time the hostname should be enough, no need the full URL endpoint

Are you convinced to merge this PR? Else, what do you advice?